applications/serial_lte_modem/src/nrfcloud/slm_at_nrfcloud.c

```diff
@@ -316,6 +322,46 @@ static int do_cloud_send_msg(const char *message, int len)
 	return err;
 }
 
+static int do_cloud_send_generic_msg(const char *message, int len)
```

Comment:
Just a note (for other reviewers) that len is in practice limited by the data buf size, which is 4K by default.

Reply:
Yes, but more important is that data after "len" is garbage

Reply:
The point was that we do not need to consider a scenario where huge messages are being send. Data after len is not garbage, but it should not be accessed either.